lib/views/stream-view.js

```diff
@@ -54,29 +82,65 @@ var StreamView = function StreamView(options) {
 
     border: "line",
     style: {
-      fg: "white",
-      bg: "black",
+      fg: this.layoutConfig.fgColor,
+      bg: this.layoutConfig.bgColor,
       border: {
-        fg: options.color || "#f0f0f0"
+        fg: this.layoutConfig.borderColor
       }
     }
   });
 
-  this.recalculatePosition();
-
   fixLogScroll(this.node);
 
+  this.recalculatePosition();
+
   this.parent.append(this.node);
-  var eventHandler = this.log.bind(this);
-  _.each(options.events, function (eventName) {
-    this.parent.screen.on(eventName, eventHandler);
-  }.bind(this));
 };
 
-StreamView.prototype = Object.create(BaseView.prototype);
+StreamView.prototype.getDefaultLayoutConfig = function () {
+  return {
+    borderColor: "#F0F0F0",
+    fgColor: "white",
+    bgColor: "black",
+    streams: ["stdout", "stderr"],
+    scrollback: 1000
```

Comment:
Yeah, I like the idea that simple params could be overridden without having to write custom code.